boar_bar/settings/local.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from django.core.exceptions import ImproperlyConfigured
+from .base import *
```

Comment:
Please do not use the asterisk for importing, it makes it unclear what you import and what you have available.